requirements.txt

```diff
@@ -1,5 +1,5 @@
 ops
-cosl
```

Comment:
This was done because 0.0.24 has breaking changes until [PR#6](https://github.com/canonical/loki-coordinator-k8s-operator/pull/6) is merged